### ISSUE TITLE
Standardise on python version (2/3) for README and related documentation

### ISSUE THREAD

**Issue by AdamHarries** [django-danceschool/django-danceschool#86]
Both the readme and other doc currently uses both python3 and python2 commands fairly interchangeably in the basic installation guide. Unfortunately, mixing together said version will result in a failing installation, even if the commands are followed correctly. 

It would be good to standardise on a version of python for all documentation, or standardise on a set of commands (e.g. `python`, `pip` rather than a mix of `python3`, `python`, `pip`, `pip3`)

**Comment by afs2015** [comment]
@AdamHarries: Good point, @leetucker do we have any dependencies that require python2? I'm inclined to go with python3 forward unless if that is an issue.

**Comment by leetucker** [comment]
@AdamHarries @afs2015  sorry for the extreme delay in getting to discuss this.  The project is explicitly Python 3.4+ at this point, and in the 0.6.0-dev branch I have removed any code that was intended for compatibility with Python 2.

I'm beginning to refactor the docs now, taking some suggestions from #85, and making explicit note of the existence of the [school template](https://github.com/django-danceschool/school-template/) repository for production deployments.  So, I can standardize everything to use the `python3` and `pip3` commands instead of `python` and `pip`.
